packages/components/splitter/src/split-bar.vue

```diff
@@ -0,0 +1,169 @@
+<script setup lang="ts">
+import { computed, ref } from 'vue'
+import { useEventListener } from '@vueuse/core'
+import {
+  ArrowDown,
+  ArrowLeft,
+  ArrowRight,
+  ArrowUp,
+} from '@element-plus/icons-vue'
+import { useNamespace } from '@element-plus/hooks'
+import type { PropType } from 'vue'
+
+const ns = useNamespace('splitter-bar')
+
+defineOptions({
+  name: 'ElSplitterBar',
+})
+
+const props = defineProps({
+  index: {
+    type: Number,
+    required: true,
+  },
+  layout: {
+    type: String as PropType<'horizontal' | 'vertical'>,
+    default: 'horizontal',
+  },
+  resizable: {
+    type: Boolean,
+    default: true,
+  },
+  startCollapsible: {
+    type: Boolean,
+    default: false,
+  },
```

Comment:
By default the boolean value is converted to false. we can remove all `default: false` for boolean props.
```suggestion
  startCollapsible: Boolean,
```